conf/locale/ru/LC_MESSAGES/djangojs.po

```diff
@@ -4724,7 +4724,19 @@ msgstr "Дополнительная информация"
 
 #: lms/static/js/student_account/views/account_settings_factory.js
 msgid "Social Media Links"
-msgstr "Ссылки на социальные сети"
+msgstr "Ссылка на профиль фейсбук"
+
+#: lms/static/js/student_account/views/account_settings_factory.js
+msgid "Enter your "
+msgstr "Введите свое имя пользователя "
+
+#: lms/static/js/student_account/views/account_settings_factory.js
+msgid " username or the URL to your "
+msgstr " или URL-адрес своей страницы "
+
+#: lms/static/js/student_account/views/account_settings_factory.js
+msgid " page. Delete the URL to remove the link."
```

Comment:
changed compared to https://github.com/raccoongang/edx-platform/pull/1294/files#diff-a60b99e24e36431c3d9c2c4ddec49062R4738